stl/inc/memory

```diff
@@ -308,10 +308,24 @@ _NoThrowFwdIt uninitialized_fill_n(_NoThrowFwdIt _First, const _Diff _Count_raw,
 }
 #endif // _HAS_IF_CONSTEXPR
 
+#if _HAS_CXX20
+template <class _Ty, class... _Types>
+auto construct_at(_Ty* const _Location, _Types&&... _Args) noexcept(noexcept(::new (const_cast<void*>(
+    static_cast<const volatile void*>(_Location))) _Ty(_STD forward<_Types>(_Args)...))) // strengthened

```

Comment:
To verify, you changed this from `is_nothrow_constructible_v<_Ty, _Types...>` to handle `indestructible`?

Reply:
Yes, after Casey explained the rationale for why the spec doesn't use `is_constructible` for that *Mandates*.